src/scss/_versionmodified.scss

```diff
@@ -0,0 +1,68 @@
+div.versionadded, div.versionchanged, div.deprecated {
+  vertical-align: middle;
+  margin: 1.5625em auto;
+  padding: 0 0.6rem 0 0.6rem;
+  overflow: hidden;
+  page-break-inside: avoid;
+  border-left: 0.2rem solid;
+  border-color: rgba(var(--pst-color-versionmodified-default), 1);
+  border-radius: 0.2rem;
+  box-shadow: 0 0.2rem 0.5rem rgba(0, 0, 0, 0.05), 0 0 0.0625rem rgba(0, 0, 0, 0.1);
+  transition: color 250ms, background-color 250ms, border-color 250ms;
```

Comment:
Hi @12rambau! I was debugging some accessibility tests and it brought me to this PR and this line of code. In order to get an accurate color contrast, I have to pause the accessibility test long enough for this transition to finish (after setting the light/dark theme). Anyway, was just wondering if this transition is needed or if we can remove it. Thanks!

Reply:
I think it's useless. From what I remember I aligned this behavior on the admonitions that were doing the same back then.